packages/serverless/ci/dind-nodejs-aws-jq/Dockerfile

```diff
@@ -18,8 +19,8 @@
 FROM node:22-alpine
 
 ENV DOCKER_CHANNEL=stable \
-    DOCKER_VERSION=20.10.11 \
-    GLIBC_VER=2.34-r0
+    DOCKER_VERSION=27.3.1 \  
```

Comment:
see for latest versions: https://docs.docker.com/engine/release-notes/27/
https://github.com/sgerrand/alpine-pkg-glibc/releases